client/src/tsconfig.json

```diff
@@ -3,7 +3,7 @@
 		"target": "es6",
 		"module": "commonjs",
 		"moduleResolution": "node",
-		"sourceMap": false,
+		"sourceMap": true,
```

Comment:
@rothfels for TS libraries it's actually common to publish .js, .d.ts and .js.map so you can step into the library while debugging